src/le.py

```diff
@@ -3593,7 +3787,10 @@ def main_root():
     }
     for cmd, func in commands.items():
         if cmd == args[0]:
-            return func(args[1:])
+            if config.multilog and cmd == 'follow':
+                return cmd_follow_multilog(args[1:])
+            else:
+                return func(args[1:])
```

Comment:
yep! noticed that and it was mentioned before I started that is was a very long file - there are other aspects that need refactoring and break out - however, as an aside, there is no body of unit tests providing coverage as a safety net - so, I was thinking at a future date to bring in pytest for both ATs and UTs on this - that in itself would be a bit of a job.